### ISSUE TITLE
NoSuchMethodError when trying to open Shell

### ISSUE THREAD

**Issue by milleruntime** [apache/accumulo#1219]
```
[centos@leader1 ~]$ accumulo shell -u root
Loading configuration from /home/centos/install/accumulo-2.0.0/conf/accumulo-client.properties
Exception in thread "shell" java.lang.NoSuchMethodError: java.nio.ByteBuffer.rewind()Ljava/nio/ByteBuffer;
	at org.apache.accumulo.core.client.security.tokens.PasswordToken.setPassword(PasswordToken.java:140)
	at org.apache.accumulo.core.client.security.tokens.PasswordToken.<init>(PasswordToken.java:61)
	at org.apache.accumulo.core.conf.ClientProperty.getAuthenticationToken(ClientProperty.java:280)
	at org.apache.accumulo.shell.Shell.config(Shell.java:326)
	at org.apache.accumulo.shell.Shell.execute(Shell.java:513)
	at org.apache.accumulo.start.Main.lambda$execKeyword$0(Main.java:129)
	at java.lang.Thread.run(Thread.java:748)
```

This was after the disk filled up running CI for 2 days.  The Master node wasn't full though, which is where I tried running the shell.  This was from the RC-0 branch so its possible I built with Java-11.  And I believe the pom builds for java 8.  

Extracting the core jar:
```
file org/apache/accumulo/core/Constants.class 
org/apache/accumulo/core/Constants.class: compiled Java class data, version 52.0 (Java 1.8)
```

**Comment by milleruntime** [comment]
OK this was definitely built using
Build-Jdk: 11.0.3
and I used mvn clean package

**Comment by milleruntime** [comment]
This is now fixed in 2.0.0-SNAPSHOT

**Comment by ctubbsii** [comment]
@milleruntime When you say it is fixed in 2.0.0-SNAPSHOT, do you mean that it is fixed when `Build-Jdk` is 11 and runtime JVM is 8?

**Comment by milleruntime** [comment]
> @milleruntime When you say it is fixed in 2.0.0-SNAPSHOT, do you mean that it is fixed when `Build-Jdk` is 11 and runtime JVM is 8?

Yes

**Comment by ctubbsii** [comment]
> > @milleruntime When you say it is fixed in 2.0.0-SNAPSHOT, do you mean that it is fixed when `Build-Jdk` is 11 and runtime JVM is 8?
> 
> Yes

Weird. Unless I'm misunderstanding something (entirely possible), then Java 11 may have produced different byte code with, and without, the `-release 8` option, even though `-source 1.8` and `-target 1.8` were specified and the animal-sniffer enforcer rule verified the compilation results against Java 1.8. That's very strange, but if it works with the `-release 8` option (which is now automatically added by a profile which is activated when building with javac 9 and later), then I guess all is well.
